wallet/src/Pos/Wallet/Web/Methods/Txp.hs

```diff
@@ -31,8 +31,12 @@ rewrapTxError
     :: forall m a. MonadCatch m
     => Text -> m a -> m a
 rewrapTxError prefix =
-    rewrapToWalletError (\SomeException{} -> True) (InternalError . sbuild) .
-    rewrapToWalletError (\TxError{} -> True) (RequestError . sbuild)
+    rewrapToWalletError (\SomeException{}     -> True) (InternalError . sbuild) .
+    rewrapToWalletError (\NotEnoughMoney{}    -> True) (RequestError . sbuild) .
+    rewrapToWalletError (\FailedToStabilize{} -> True) (RequestError . sbuild) .
```

Comment:
It doesn't work like you may expect.
E.g. `Nothing & \Just 5 -> ()` would throw error.
Here you have to specify exception type without mentioning constructors

Reply:
@Martoon-00 how would you suggest doing that?

Reply:
Something like
`rewrapToWalletError @TxError (\(e :: TxError) -> True) (...)`
or perhaps more prettier
`rewrapToWalletError @TxError (const True) (...)`